libtenzir/builtins/aggregation-functions/once.cpp

```diff
@@ -0,0 +1,112 @@
+//    _   _____   __________
+//   | | / / _ | / __/_  __/     Visibility
+//   | |/ / __ |_\ \  / /          Across
+//   |___/_/ |_/___/ /_/       Space and Time
+//
+// SPDX-FileCopyrightText: (c) 2025 The Tenzir Contributors
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <tenzir/arrow_table_slice.hpp>
+#include <tenzir/fbs/aggregation.hpp>
+#include <tenzir/flatbuffer.hpp>
+#include <tenzir/tql2/eval.hpp>
+#include <tenzir/tql2/plugin.hpp>
+
+namespace tenzir::plugins::once {
+
+namespace {
+
+class once_instance final : public aggregation_instance {
+public:
```

Comment:
Do we keep this intentionally undocumented because it's only use is in `chart2` right now?

Reply:
Yes, that was the intention as it's not generally useful.